cvrf2csaf/section_handlers/vulnerability.py

```diff
@@ -14,6 +15,76 @@ def __init__(self, config):
     def _process_mandatory_elements(self, root_element):
         pass
 
+    @staticmethod
+    def _handle_involvements(root_element):
+        # TODO: potential handling of missing `date` entry
+        involvements = []
+        for involvement_elem in root_element.Involvement:
+            involvement = {'party': involvement_elem.attrib['Party'].lower(),
+                           'status': involvement_elem.attrib['Status'].lower().replace(' ', '_')}
+
+            if hasattr(involvement_elem, 'Description'):
+                involvement['summary'] = involvement_elem.Description.text
+            involvements.append(involvement)
+
+        return involvements
+
+    @staticmethod
+    def _handle_product_statuses(root_element):
+        statuses = defaultdict(list)
+        for status_elem in root_element.Status:
+            status_type = status_elem.attrib['Type'].lower().replace(' ', '_')
+            product_ids = [product_id.text for product_id in status_elem.ProductID]
+            statuses[status_type].extend(product_ids)
+
+        return statuses
+
+    @staticmethod
+    def _handle_threats(root_element):
+        threats = []
+        for threat_elem in root_element.Threat:
+            threat = {'details': threat_elem.Description.text,
+                      'category': threat_elem.attrib['Type'].lower().replace(' ', '_')}
+
+            if hasattr(threat_elem, 'ProductID'):
+                threat['product_ids'] = [product_id.text for product_id in threat_elem.ProductID]
+
+            if hasattr(threat_elem, 'GroupID'):
+                threat['group_ids'] = [group_id.text for group_id in threat_elem.GroupID]
+
+            if 'Date' in threat_elem.attrib:
+                threat['date'] = threat_elem.attrib['Date']
+
+            threats.append(threat)
+
+        return threats
+
+    @staticmethod
+    def _handle_remediations(root_element):
+        remediations = []
+        for remediation_elem in root_element.Remediation:
+            remediation = {'category': remediation_elem.attrib['Type'].lower().replace(' ', '_'),
```

Comment:
`Will Not Fix` was renamed to `no_fix_planned`.

Reply:
please fix no_fix_planned.